### ISSUE TITLE
Trace data showing incorrect times

### ISSUE THREAD

**Issue by adjohn** [iopipe/iopipe-python#144]
> The python traces say duration was days, any idea why?
![screen shot 2018-01-29 at 9 17 13 pm](https://user-images.githubusercontent.com/168944/35544895-9d21793a-053a-11e8-8768-cdac1bd6de6f.png)

**Link:** [Conversation with Customer](https://app.intercom.com/a/apps/jrzvdthw/respond/inbox/all/conversations/14347076635)

___
Intercom Operator will send a note to [this conversation](https://app.intercom.com/a/apps/jrzvdthw/respond/inbox/all/conversations/14347076635) when this issue is closed.

**Comment by coreylight** [comment]
The `duration` values on each mark seem to be very high.
```json
{
  "data": {
    "invocations": {
      "size": 30,
      "from": null,
      "total": 1,
      "functionId": "xxx",
      "ruleResult": false,
      "rules": [],
      "invocations": [
        {
          "timestamp": 1517278018490,
          "id": "xxx",
          "duration": [
            2574.63
          ],
          "version": "$LATEST",
          "memory": {
            "tier": 1024,
            "free": 3444016,
            "total": 3855608,
            "rss": 31.8,
            "rssPercentile": null,
            "percentage": [
              {
                "label": "0",
                "value": 4
              }
            ]
          },
          "cpu": {
            "idle": 99.81,
            "irq": 0,
            "sys": 0.07,
            "user": 0.13,
            "nice": 0
          },
          "errors": [
            {
              "name": null,
              "stack": null,
              "message": null,
              "stackHash": null,
              "count": 1
            }
          ],
          "agent": {
            "version": "1.0.0"
          },
          "runtime": {
            "name": "Python",
            "version": "3.6.1"
          },
          "performanceEntries": [
            {
              "entryType": "measure",
              "startTime": 47186,
              "name": "measure:iopipe",
              "duration": 2575486118,
              "timestamp": 1517278018489
            },
            {
              "entryType": "measure",
              "startTime": 1698966,
              "name": "measure:fetch security scans",
              "duration": 247478533,
              "timestamp": 1517278018491
            },
            {
              "entryType": "measure",
              "startTime": 249207207,
              "name": "measure:fetch scan",
              "duration": 864462729,
              "timestamp": 1517278018739
            },
            {
              "entryType": "measure",
              "startTime": 1129260875,
              "name": "measure:fetch metadata",
              "duration": 1204704294,
              "timestamp": 1517278019619
            },
            {
              "entryType": "measure",
              "startTime": 2334003180,
              "name": "measure:event publish",
              "duration": 240588393,
              "timestamp": 1517278020823
            }
          ],
          "plugins": [
            {
              "name": "trace",
              "enabled": true,
              "uploads": null
            }
          ],
          "coldStart": true,
          "customMetrics": [
            {
              "name": "event_id",
              "n": null,
              "s": "xxx"
            }
          ],
          "errorCount": 0,
          "provider": {
            "requestId": "xxx",
            "arn": "xxx",
            "logStreamName": "xxx",
            "logGroupName": "xxx"
          }
        }
      ]
    }
  }
}
```

**Comment by kolanos** [comment]
This bug was fixed in v1.0.1.
